setup.py

```diff
@@ -15,7 +15,7 @@
     package_data={'bluesky': ['schema/*.json']},
     install_requires=['jsonschema', 'traitlets', 'cycler',
                       'numpy', 'matplotlib', 'super_state_machine',
-                      'historydict'],
+                      'historydict', 'event_model'],
```

Comment:
Hmm. Then why do the tests pass? I think `event_model` is correct -- we want the Python package name, not the conda package name, and `event-model` is not a valid Python package name. This is confusing, and I'm not 100% sure I'm right about this, but I were wrong I would expect the Travis build to fail when bluesky is installed.